archivesunleashed/auk
# Cleaning up documentation prose

- Cleans up tense, repetitive text, adds paragraphs
- Enhances usability

# What does this Pull Request do?

One of our project RAs (Rebecca MacAlpine) went through the website and made stylistic suggestions throughout as a novel user. I've taken her suggestions and implemented them (more or less) in this pull request. 

They're mostly surface changes but I think they do a good job of adding some more polish to the site.

# How should this be tested?

A description of what steps someone could take to:
* TravisCI turns green
* Review prose changes to make sure they all make sense

# Interested parties

@SamFritz @ruebot
